l10n_ro_config/data/siruta/2res.country.state.csv

```diff
@@ -0,0 +1,43 @@
+id,zone_id/id,siruta
+l10n_ro.RO_AB,RO_C,10
```

Comment:
Județele nu mai sunt in l10n_ro ci in base.

Reply:
Le updatam când migram siruta.